.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] You seem to have committed node_modules, kindly avoid committing it. Please ensure node_modules isn't part of the committed files.